kg/api.py

```diff
@@ -7,14 +7,14 @@
 client = Neo4jClient()
 
 
-@app.route("/search", methods=["GET"])
+@app.route("/v1/node", methods=["GET"])
```

Comment:
Is the `node` portion of the endpoint name appropriate  here? Or is this a temporary name as we have not discussed the UI integration (i.e. what the return value should look like)?

Reply:
The return data structure is a bit more complicated (not just a node / list of nodes), a better endpoint name is `alerts` since fundamentally it searches into a set of outbreak alerts. I'll use that for now.